rust/rbac-registration/src/utils/hashing.rs

```diff
@@ -2,14 +2,6 @@
 
 use blake2b_simd::{self, Params};
 
-/// Convert the given value to `blake2b_244` array.
-pub(crate) fn blake2b_244(value: &[u8]) -> anyhow::Result<[u8; 28]> {
-    let h = Params::new().hash_length(28).hash(value);
-    let b = h.as_bytes();
-    b.try_into()
-        .map_err(|_| anyhow::anyhow!("Invalid length of blake2b_244, expected 28 got {}", b.len()))
-}
-
 /// Convert the given value to `blake2b_256` array.
 pub(crate) fn blake2b_256(value: &[u8]) -> anyhow::Result<[u8; 32]> {
```

Comment:
I think the hashing function is available in other cat-lib crate, need to recheck

Reply:
Yes, its generalized into another crate, shouldn't be here anymore.